front/pages/api/v1/w/[wId]/assistant/agent_configurations.ts

```diff
@@ -0,0 +1,53 @@
+import { NextApiRequest, NextApiResponse } from "next";
+
+import { getAgentConfigurations } from "@app/lib/api/assistant/configuration";
+import { Authenticator, getAPIKey } from "@app/lib/auth";
+import { ReturnedAPIErrorType } from "@app/lib/error";
+import { apiError, withLogging } from "@app/logger/withlogging";
+import { GetAgentConfigurationsResponseBody } from "@app/pages/api/w/[wId]/assistant/agent_configurations";
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<
+    GetAgentConfigurationsResponseBody | ReturnedAPIErrorType | void
+  >
+): Promise<void> {
+  const keyRes = await getAPIKey(req);
+  if (keyRes.isErr()) {
+    return apiError(req, res, keyRes.error);
+  }
+
+  const { auth, keyWorkspaceId } = await Authenticator.fromKey(
+    keyRes.value,
+    req.query.wId as string
+  );
+
+  if (!auth.isBuilder() || keyWorkspaceId !== req.query.wId) {
```

Comment:
can be user

Reply:
even if not relevant/possible right now so I guess no-op

Reply:
API key == builder no ?